src/utils/redis.ts

```diff
@@ -1,16 +1,25 @@
 import * as redis from "redis";
-import RedisClustr = require("redis-clustr");
 import * as appInsights from "applicationinsights";
 import { pipe } from "fp-ts/lib/function";
 import * as RA from "fp-ts/lib/ReadonlyArray";
 import * as O from "fp-ts/lib/Option";
 import { keyPrefixes } from "../services/redisSessionStorage";
 import { log } from "./logger";
 
-export function createSimpleRedisClient(redisUrl?: string): redis.RedisClient {
+export async function createSimpleRedisClient(
+  redisUrl?: string
+): Promise<redis.RedisClientType> {
   const redisUrlOrDefault = redisUrl || "redis://redis";
   log.info("Creating SIMPLE redis client", { url: redisUrlOrDefault });
-  return redis.createClient(redisUrlOrDefault);
+  const redisClient = redis.createClient<
+    redis.RedisDefaultModules,
+    Record<string, never>,
+    Record<string, never>
+  >({
+    url: redisUrlOrDefault
```

Comment:
The variable is now an host. Must be handled as url instead to avoid an exception